lint.js

```diff
@@ -68,7 +68,7 @@ More information: https://speccy.io/rules/1-rulesets#${rule.name}
 const command = async (specFile, cmd) => {
     config.init(cmd);
     const jsonSchema = config.get('jsonSchema');
-    const verbose = config.get('quiet') ? 0 : (config.get('verbose') ? 2 : 1);
+    const verbose = config.get('quiet') ? 0 : (config.get('verbose') ? config.get('verbose') : 1);
```

Comment:
Makes sense 👍

Reply:
Couldn't you use `config.get(key, defaultValue)`?

Something like this:

```javascript
config.get('verbose', 1)
```
instead of this part:

```javascript
(config.get('verbose') ? config.get('verbose') : 1)
```

Reply:
@mdaloia makes a good point!